### DISCUSSION TITLE
How to add new vocabulary to menu vocabulary?

### DISCUSSION THREAD

**Discussion by vankhanh0911** [ZuodaoTech/everyone-can-use-english#653] [Q&A] [answered]
How to add new vocabulary to menu vocabulary?

**Comment by an-lee** [comment, marked as answer]
The vocabulary feature is in super early state. Fow now, only the words lookedup by AI Dictionary will be added to the vocabulary.

**Reply by vankhanh0911** [reply]
Thank you.
